README.md

```diff
@@ -398,8 +398,8 @@ If you have issues with the APIs or have a question about the Watson services, s
 
 Tested on:
 
-* MRI Ruby (RVM): 2.3.7, 2.4.4, 2.5.1
```

Comment:
Maybe we leave a warning for now that support for old versions will be deprecated in next major release.

Reply:
good call